### PR TITLE
ECS-390: add slider control type property.

### PR DESCRIPTION
**Pull request by wwsno**
Requires configuring default value, minValue, maxValue and stepSize.

### CODE REVIEW

File: lib/validators/default-values-validator.ts [hunk 1]
```diff
@@ -90,10 +90,16 @@ export class DefaultValuesValidator extends Validator {
         return true;
     }
 
+    private validateNumberValue(property: ComponentProperty): boolean {
```
Comment (wwjhu) [inline lib/validators/default-values-validator.ts:93]: Should this be named `validateDefaultValue`?

**Review by wwjhu — APPROVED** [review]
Two minor thingies.

File: lib/validators/default-values-validator.ts (hunk 1 above)
Comment (wwsno) [reply lib/validators/default-values-validator.ts:93]: Hmm, I think it is already clear this is about default values? This validates the default value is a number. The other two functions validate if they are a string or object. It depends on the control type which values are supported.

Comment (wwjhu) [reply lib/validators/default-values-validator.ts:93]: The way I read the function name is that it validates that a value is a number while in reality it validates that the default value is a number. But that same goes for the other functions that I didn't see, so let's leave it like this.

**Pull request merged**